Readme.md

```diff
@@ -1,28 +1,40 @@
-* <a href="https://documentation.devexpress.com/#WPF/DevExpressXpfDockingDockLayoutManager_AllowMergingAutoHidePanelstopic">DockLayoutManager.AllowMergingAutoHidePanels</a>.
-* <strong>MergeType</strong> properties (they are available in <a href="https://documentation.devexpress.com/WPF/DevExpressXpfRibbonRibbonPage_MergeTypetopic.aspx">RibbonPage</a>, <a href="https://documentation.devexpress.com/WPF/DevExpressXpfRibbonRibbonPageGroup_MergeTypetopic.aspx">RibbonPageGroup</a> and <a href="https://documentation.devexpress.com/WPF/DevExpressXpfBarsBarItemLinkBase_MergeTypetopic.aspx">BarItemLinkBase</a>) allow you to control how items with similar captions will be merged.
-* <strong>MergeOrder</strong> properties (they are available in <a href="https://documentation.devexpress.com/WPF/DevExpressXpfRibbonRibbonPageCategoryBase_MergeOrdertopic.aspx">RibbonPageCategoryBase</a>, <a href="https://documentation.devexpress.com/WPF/DevExpressXpfRibbonRibbonPage_MergeOrdertopic.aspx">RibbonPage</a>, <a href="https://documentation.devexpress.com/WPF/DevExpressXpfRibbonRibbonPageGroup_MergeOrdertopic.aspx">RibbonPageGroup</a> and <a href="https://documentation.devexpress.com/WPF/DevExpressXpfBarsBarItemLinkBase_MergeOrdertopic.aspx">BarItemLinkBase</a>) can be used to control the merging order.
-* <a href="https://documentation.devexpress.com/#WPF/DevExpressXpfRibbonRibbonControl_SelectedPageOnMergingtopic">RibbonControl.SelectedPageOnMerging</a> sets which ribbon's page will be selected when ribbons are merged.<br><br>For more information, refer to <a href="https://documentation.devexpress.com/#WPF/CustomDocument10587">Merging Overview</a>.</p>
-
-<br/>
-
-
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T304338)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+# WPF Dock Layout Manager - Automatically Merge Ribbon Controls
+
+The following controls support automatic bar merging:
+
+* [DXTabControl](https://docs.devexpress.com/WPF/7975/controls-and-libraries/layout-management/tab-control/fundamentals/dxtabcontrol)
+* [Dock Layout Manager](https://docs.devexpress.com/WPF/6191/controls-and-libraries/layout-management/dock-windows)
+  
+In this example, the [docklayoutmanager](https://docs.devexpress.com/WPF/DevExpress.Xpf.Docking.DockLayoutManager) component implements the tabbed MDI interface. The main application window and tabs contain ribbon controls.
+
+The example merges a pages, page groups, and items from child ribbons to the main ribbon control.

```

Comment:
Can you update this line as follows?

`The example merges pages, page groups, and items from child ribbons to the main ribbon control.`